src/org/rascalmpl/library/Location.rsc

```diff
@@ -43,6 +44,18 @@ loc relativize(list[loc] haystack, loc needle) {
     }
 }
 
+@synopsis{Convert Windows path syntax to a `loc` value}
+@description{
+This conversion supports generic Windows path syntax, including:
+* Absolute drive-specific: `C:\Program Files`
+* Relative drive-specific: `C:hello.txt`
+* Relative: `hello.txt`
+* Directory-relative: `\hello.txt`
+* UNC format: `\\\\system07\\C$\\`
+
+Windows paths, againat popular believe, support both `/` and `\` as path separators.
```

Comment:
`s/againat/against/`

apart from that, yes, it has always been supported. and you can even mix them..